src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Mobile/UnoQuickStart.Mobile.csproj

```diff
@@ -26,6 +26,7 @@
 		<ApplicationDisplayVersion>1.0</ApplicationDisplayVersion>
 		<ApplicationVersion>1</ApplicationVersion>
 
+		<AndroidManifest>Android\AndroidManifest.xml</AndroidManifest>
```

Comment:
This is a symptom of another problem, generally caused by nuget packages not being restored properly.

Reply:
@jeromelaban NuGet needs a passing design-time build in order to restore, and the error about AndroidManifest happens in design-time builds by a target that is run before anything from NuGet runs. So, AndroidManifest needs to be set early.

Reply:
I assume same adjustment will be needed in the new templates as well (in Uno Extensions)